clustered/clustered-dist/src/assemble/server/conf/cluster.cfg

```diff
@@ -5,14 +5,9 @@ failover-priority=availability
 offheap-resources=main:512MB
 stripe-names=default-stripe
 default-stripe:node-names=default-node
-default-node:node-bind-address=0.0.0.0
-default-node:node-group-bind-address=0.0.0.0
-default-node:node-group-port=9430
-default-node:node-hostname=localhost
-default-node:node-log-dir=%H/terracotta/logs
-default-node:node-logger-overrides=
-default-node:node-name=default-server
-default-node:node-port=9410
-default-node:node-public-hostname=
-default-node:node-public-port=
-default-node:tc-properties=
+default-node:bind-address=0.0.0.0
```

Comment:
These changes should really be in a separate commit. Arguably they should be daggy... but the timegap is short so it's probably harmless.